test/system/helpers.bash

```diff
@@ -733,6 +733,20 @@ function is_aarch64() {
     [ "$(uname -m)" == "aarch64" ]
 }
 
+function is_rhel_or_centos() {
+    if find /etc/redhat-release && grep -Eiq "Red Hat Enterprise Linux|CentOS Stream" /etc/redhat-release; then
+        return 0
+    fi
+    return 1
+}
+
+function is_rawhide() {
+    if find /etc/fedora-release && grep -Eiq "Rawhide" /etc/fedora-release; then
+        return 0
+    fi
+    return 1
+}
```

Comment:
Should this be cached? Also hard coded to redhat/fedora-release seems unnecessary.
I would think the best way would be to parse `/etc/os-release` and then safe ID and VERSION_ID fields like done in e2e tests, i.e. GetHostDistributionInfo()?